lib/plugins/interactiveCli/index.js

```diff
@@ -11,7 +11,11 @@ module.exports = class InteractiveCli {
 
     const { processedInput } = serverless;
     if (processedInput.commands.length) return;
-    if (!_.isEmpty(processedInput.options)) return;
+    const options = new Set(keys(processedInput.options));
+    options.delete('name');
+    options.delete('app');
+    options.delete('org');
```

Comment:
We need to precisely clarify scope of this options, and agree on how to handle them in cases where they do not apply, or are controversial to apply.

Having above, as in case of other CLI commands, we need to prepare some help that documents those options.

To save time, I propose that we hold any implementation, until all details are clarified and agreed

Reply:
I thought they _exclusively_ apply to the interactive cli. right @skierkowski ?

Reply:
> I thought they exclusively apply to the interactive cli. right

Yes, I meant scope of this options in interactive CLI.. as there are different cases (services not applicable for dashboard, existing and already fully configured services, users logged in with different dashboard accounts than ones for which options were copied etc.)